release-testing-utils/README.rst

```diff
@@ -0,0 +1,3 @@
+This is a place to put course exports or other helpful resources for executing named release tests.
+
+When adding a file here, please clearly identify the test case(s) for which the file is useful.
```

Comment:
Could you add a note about what this course you're adding is useful for? :)

Reply:
The name of the file is tc-devops-16.tar.gz, I figured that would suffice? That way this readme can't fall out of date.

Yes, a waffle flag is needed. I was hoping that the test instructions would stay in the test sheet, and that this would just be an artifact that someone can grab in order to skip some setup steps. i.e.:

> SETUP
> * Flip waffle X
> * Create a new course
> * Download and import (link to this tar) OR follow these steps to manually prepare the course using these steps:
>   * ... original setup steps

Reply:
gotcha, there's always the risk that the test accidentally gets deleted or renamed but if you think that's not a risk, we can merge

Reply:
@sarina I edited the README to hopefully be a little clearer, LMK what you think :)

In theory, the unique test IDs shouldn't change, even if the test is renamed. They're supposed to be stable identifiers that we can compare release to release.